synapse/rest/client/v1/admin.py

```diff
@@ -25,6 +26,34 @@
 logger = logging.getLogger(__name__)
 
 
+class UsersRestServlet(ClientV1RestServlet):
+    PATTERNS = client_path_patterns("/admin/users/(?P<user_id>[^/]*)")
```

Comment:
is my understanding correct? this takes a user_id, and then ignores it? @morteza-araby

Reply:
This is now tracked as https://github.com/matrix-org/synapse/issues/2522